### ISSUE TITLE
Having the flags always visible 

### ISSUE THREAD

**Issue by zouhair** [bastimeyer/livestreamer-twitch-gui#106]
Hi,

it would be great if, when browsing streams, to have flags always visible not only after a mouse hover.

Much easier on the mouse travel.

Thanks

**Comment by bastimeyer** [comment]
Hi...
Well, I don't think this is a great idea. This would cause too much distraction in a stream list. There is a reason why the flags are hidden without hovering the element. It's the same with the game being played. Showing too much at the same time renders a single information pretty much useless.

![image](https://cloud.githubusercontent.com/assets/467294/7962212/79073d18-0a0c-11e5-9bb1-5f6b37df89fa.png)

If you're looking for a stream in a specific language, then I would agree that it's not ideal, since you'd have to hover all the streams. But what's actually missing in this situation is a language search feature (the twitch API doesn't provide this functionality, though) or a filtering system.

**Comment by zouhair** [comment]
It feels like a matter of preferences then. I don't feel it is cluttered at all but extremely useful for me.

I'm OK with hidden game being played as I usually browse by game rather than just random streams.

It would be nice to have as a toggle non-default option.though.

Thanks
